core/services/ping/ping360_driver.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from shutil import which
+from subprocess import Popen
+from typing import Optional
+
+import serial
+
+from pingdriver import PingDriver
+from pingutils import PingDeviceDescriptor
+
+
+class Ping360Driver(PingDriver):
+    def __init__(self, ping: PingDeviceDescriptor, port: int) -> None:
+        super().__init__(ping, port)
+        self.process: Optional[Popen[bytes]] = None
+        self.baudrate: int = 2000000
+
+    def set_low_latency_and_high_baud(self) -> None:
+        """
+        sets the latency_timer for the device and attemps to set the ping360 to a 2M baudrate
+        """
+
+        # Open device with 2M baud to setup Ping360
+        logging.info("Setting baud to 2M...")
+        ser = serial.Serial(self.ping.port.device, self.baudrate)
+        ser.send_break()
+        ser.write("UUUUUUU".encode())
+        ser.close()
+        target_device = self.ping.port.device
+        device_name = target_device.strip().split("/")[-1]
+        latency_file = "/sys/bus/usb-serial/devices/{0}/latency_timer".format(device_name)
+        logging.info(f"Latency file: {latency_file}")
+
+        with open(latency_file, "w") as p:
+            p.write("1")
+            p.flush()
+
+    def start(self) -> None:
+        self.set_low_latency_and_high_baud()
+        bridges = which("bridges")
+        command_line = f"{bridges} -u 0.0.0.0:{self.port} -p {self.ping.port.device}:{self.baudrate}"
+        logging.info(f"launching {command_line}")
+        # pylint: disable=consider-using-with
+        self.process = Popen(command_line.split(" "))
```

Comment:
You should use `shlex.split`, the result of split(" ") is different, since it may break with multiple arguments per flag